tests/test_base_resources.py

```diff
@@ -441,6 +442,28 @@ def test_ep(self, reqmock):
         assert tm[0].num == 1.0
         assert tm[1].num == 2.0
 
+    def test_ep_auto_retry_true(self, reqmock):
+        responses = [
+            {'status_code': 503, 'headers': {'Retry-After': '0.01'}},
```

Comment:
done in https://github.com/analyzere/analyzere-python/pull/15/commits/9d798676c87158b0b21571ccef9b31c9c4fbff83